content/academy/web_scraping_for_beginners/data_collection/browser_devtools.md

```diff
@@ -12,15 +12,15 @@ Even though DevTools stands for developer tools, everyone can use them to inspec
 
 ## [](#elements) Elements tab
 
-When you first open Chrome DevTools, you will start on the Elements tab (In Firefox it's called the Inspector). You can use the tab to inspect the page's HTML on the left and also its CSS on the right. The items in the HTML view are called elements and they are all enclosed in HTML tags. For example `<div>`, `<p>`, and `<span>` are tags. When you add something inside of those tags, like `<p>Hello!</p>` you create an element. You can also see elements inside elements in the **Elements** tab. This is called nesting and it gives the page its structure.
+When you first open Chrome DevTools, you will start on the Elements tab (In Firefox it's called the **Inspector**). You can use this tab to inspect the page's HTML on the left hand side, and its CSS on the right. The items in the HTML view are called **elements** or **nodes**. Each element is enclosed in an HTML tag. For example `<div>`, `<p>`, and `<span>` are all tags. When you add something inside of those tags, like `<p>Hello!</p>` you create an element. You can also see elements inside other elements in the **Elements** tab. This is called nesting, and it gives the page its structure.
 
-At the bottom there's the JavaScript console which you can use to manipulate the website. If the Console is not there, you can press **ESC** to toggle it. At first it looks super complicated, but don't worry, there's no need to understand everything.
+At the bottom, there's the **JavaScript console**, which is a powerful tool which can be used to manipulate the website. If the console is not there, you can press <kbd>ESC</kbd> to toggle it. All of this might look super complicated at first, but don't worry, there's no need to understand everything just yet - we'll be walking you through all the important things you need to know.
 
 ![Chrome DevTools with elements tab and console]({{@asset web_scraping_for_beginners/data_collection/images/browser-devtools.webp}})
```

Comment:
I would get rid of the alexa screenshots even here. The website will not exist soon. Maybe we can use screenshots from the Academy itself here.

Reply:
I will do new images on Fakestore, and edit in the red circles as well. It's good for them to get somewhat familiar with it before diving in. Kinda like we did with Alexa